Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -1244,6 +1245,12 @@ void MainWindow::ShowControllersWindow()
   m_controllers_window->activateWindow();
 }
 
+void MainWindow::ShowScriptingWidget()
+{
+  static Settings& settings = Settings::Instance();
```

Comment:
Why do we have the `static` keyword here? Doesn't this make a static reference?

Reply:
Lol I think I saw the function declaration was `static Settings& Instance()` so I thought the return type had to be static therefore. I must have been sleepy!!!